packages/ethereum-contracts/package.json

```diff
@@ -47,6 +47,7 @@
     },
     "devDependencies": {
         "@decentral.ee/web3-helpers": "0.3.5",
+        "@ethersproject/contracts": "^5.0.10",
```

Comment:
do we check compatible ethers.js versions in Framework initialization?

Reply:
Support for using ethers in the  ethereum-contracts deployment script is incomplete. I don't plan to release docs for using the scripts with ethers. 

Some of the changes I made to `ethereum-contracts` were required, and others are just getting prepared for when the time comes.

So no need to perform this check here

As far as performing this check in the js-sdk, I think we don't need to do it for now. If we see issues then we can start to think about it more

Reply:
okay